styles/legacy.styles.ts

```diff
@@ -27,6 +27,18 @@ export const Heading = styled.h1`
   font-style: normal;
 `;
 
+export const SubHeading = styled.p`
+  margin: 20px 100px;
```

Comment:
Are you sure you need 100px as left and right margin?
Also you've to add breakpoints for this. Otherwise, the margin will be the same in mobile view too which we don't want.
<img width="369" alt="Screenshot 2021-11-27 at 2 32 54 PM" src="https://user-images.githubusercontent.com/56754747/143675130-d9c20a52-9b4f-4216-914b-9ed2316f7022.png">

Reply:
mobile view margin: 20 px, 20px;
web margin:20px, 40 px;